src/pointer-analysis/value_set_fi.cpp

```diff
@@ -1016,24 +1007,22 @@ void value_set_fit::assign(
         }
         else if(rhs.id()==ID_with)
         {
-          assert(rhs.operands().size()==3);
-
-          // see if op1 is the member we want
-          const exprt &member_operand=rhs.op1();
+          // see if this is the member we want
+          const exprt &member_operand = to_with_expr(rhs).where();
```

Comment:
It might be better to make a variable which is a reference to `to_with_expr(rhs)`